saleor/graphql/core/scalars.py

```diff
@@ -163,3 +164,19 @@ class Minute(graphene.Int):
 
 class Day(graphene.Int):
     """The `Day` scalar type represents number of days by integer value."""
+
+
+class PositiveInteger(graphene.Int):
+    """Positive Int custom field.
+
+    Should use in places where int value must be nonnegative (0 or greater).
+    """
+
+    @staticmethod
+    def parse_literal(ast):
```

Comment:
Why do we only implement `parse_literal` method? We currently have almost similar problem with `PositiveDecimal` scalar (ref: https://github.com/saleor/saleor/issues/14373). It may be worth investigating if that's correct approach before we add new Scalar.

Reply:
I the issue `serialize` method is mentioned, I couldn't find it in the inheritance hierarchy, from Int -> Scalar -> BaseType, UnmountedType -> OrderedType.
As far I've seen we need to implement custom serialize method in case of Decimal values, never needed one for Int types.

Reply:
`serialize` and `parse_value` in `graphene.Int` have implementation, to be more exact, it's `coerce_int` method from `graphene.Int`